src/main/java/com/jcabi/aspects/aj/MethodCacher.java

```diff
@@ -351,12 +362,22 @@ public boolean expired() {
         public boolean asyncUpdate() {
             return this.async;
         }
+
+        /**
+         * Soft reference to cached object.
+         * Visible only for testing. Don not use directly.
+         * @return Soft reference to cached object.
+         */
```

Comment:
@nesterov-n however I understand it is pretty necessary here. Can we at least name this method like `cached()` so that it doesn't resemble getter?